src/System.Collections.Immutable/tests/System.Collections.Immutable.Tests.csproj

```diff
@@ -124,5 +125,8 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
```

Comment:
You shouldn't need to add references at all for test projects since they add references automatically so please remove this itemgroup.